README.md

```diff
@@ -118,7 +165,7 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+An ERD is a relationship diagram that helps developers better organize classes by how they relate to each other. For example, a person can have a lamp(one to many). A lamp can have a single genie(one to one). And any number of people can Pet any number of lamps (many to many)
```

Comment:
👍 describes the relationships in the database which through an ORM is reflected in class organization. Can generally say an ERD defines the domain model